.gitignore

```diff
@@ -141,3 +141,7 @@ dmypy.json
 # vs code
 .vscode
 
+# testing files
+test-decanter.ipynb
```

Comment:
I think those 3 files are your test file. These files won't apprear in developers' env.